Week-13/url-shortener/src/components/Footer.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
```

Comment:
create folder called Layout in components and move Footer , headerand Navbar in it